infra/feast-operator/test/e2e/e2e_test.go

```diff
@@ -17,217 +17,38 @@ limitations under the License.
 package e2e
 
 import (
-	"fmt"
-	"os"
-	"os/exec"
-	"time"
-
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-
 	"github.com/feast-dev/feast/infra/feast-operator/test/utils"
-)
-
-const (
-	feastControllerNamespace = "feast-operator-system"
-	timeout                  = 2 * time.Minute
-	controllerDeploymentName = "feast-operator-controller-manager"
-	feastPrefix              = "feast-"
+	. "github.com/onsi/ginkgo/v2"
 )
 
 var _ = Describe("controller", Ordered, func() {
-	BeforeAll(func() {
-		_, isRunOnOpenShiftCI := os.LookupEnv("RUN_ON_OPENSHIFT_CI")
-		if !isRunOnOpenShiftCI {
-			By("creating manager namespace")
-			cmd := exec.Command("kubectl", "create", "ns", feastControllerNamespace)
-			_, _ = utils.Run(cmd)
-
-			var err error
-			// projectimage stores the name of the image used in the example
-			var projectimage = "localhost/feast-operator:v0.0.1"
-
-			By("building the manager(Operator) image")
-			cmd = exec.Command("make", "docker-build", fmt.Sprintf("IMG=%s", projectimage))
-			_, err = utils.Run(cmd)
-			ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-			By("loading the the manager(Operator) image on Kind")
-			err = utils.LoadImageToKindClusterWithName(projectimage)
-			ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-			// this image will be built in above make target.
-			var feastImage = "feastdev/feature-server:dev"
-			var feastLocalImage = "localhost/feastdev/feature-server:dev"
-
-			By("building the feast image")
-			cmd = exec.Command("make", "feast-ci-dev-docker-img")
-			_, err = utils.Run(cmd)
-			ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-			By("Tag the local feast image for the integration tests")
-			cmd = exec.Command("docker", "image", "tag", feastImage, feastLocalImage)
-			_, err = utils.Run(cmd)
-			ExpectWithOffset(1, err).NotTo(HaveOccurred())
-
-			By("loading the the feast image on Kind cluster")
-			err = utils.LoadImageToKindClusterWithName(feastLocalImage)
-			ExpectWithOffset(1, err).NotTo(HaveOccurred())
+	featureStoreName := "simple-feast-setup"
+	feastResourceName := utils.FeastPrefix + featureStoreName
+	feastK8sResourceNames := []string{
+		feastResourceName + "-online",
+		feastResourceName + "-offline",
+		feastResourceName + "-ui",
+	}
```

Comment:
just want make sure this is correct, we have `online store/server` default now. so need registry here instead of online. WDYT ?
cc @tchughesiv

Reply:
or, just add `feastResourceName + "-registry",`

Reply:
> just want make sure this is correct, we have `online store/server` default now. so need registry here instead of online. WDYT ? cc @tchughesiv

I used the old e2e tests without changing them and in them there is special care of registry, i didn't want to mix updating/fixing e2e tests following latests changes with doing an upgrade test, I think it should be a different task

Reply:
> maybe try to change `feastResourceName + "-online"` -> `feastResourceName + "-registry" ` and see where it breaks ? if this required more works then we can open a separate PR for this.

It causes the tests to fail, I think it should be in a different task

Reply:
i think we need a different method maybe? one that checks for remote registry and one that checks for local registry service. when remote registry is configured, a svc won't exist for it

Reply:
> i think we need a different method maybe? one that checks for remote registry and one that checks for local registry service. when remote registry is configured, a svc won't exist for it

Do you think it should be fixed as part of the upgrade tests ? IMO it should be a separate task - "revise existing e2e test and enhance them"

Reply:
+1

Reply:
Opened an new issues to track this https://github.com/feast-dev/feast/issues/5091